.github/workflows/e2e-tests.yaml

```diff
@@ -117,6 +117,14 @@ jobs:
         with:
           go-version: 1.21.8
 
+      - name: Add repository to Git safe directory list
+        run: git config --global --add safe.directory C:/actions-runner/_work/devpod/devpod
```

Comment:
think we don't need this anymore, right?

Reply:
ah yeah, I've started debugging it in this PR. cleaning up 👍